### ISSUE TITLE
MultiStateCheckbox: A form field element has neither an id nor a name attribute

### ISSUE THREAD

**Issue by KubaGorka** [primefaces/primereact#8293]
### Describe the bug

The MultiStateCheckbox component fails accessibility requirements.
A form field element has neither an id nor a name attribute. This might prevent the browser from correctly autofilling the form.

### System Information

```Shell
primereact: 10.9.7
```

### Steps to reproduce the behavior

Go to https://primereact.org/multistatecheckbox/

Inspect the input element of the MultiStateCheckbox

Notice that the input is marked by the browser as a node with an issue.

### Expected behavior

Input should have a name or id, so it follows the accessibility guidelines
